APIKit/APIKit.swift

```diff
@@ -115,6 +115,10 @@ public class API: NSObject, NSURLSessionDelegate, NSURLSessionDataDelegate {
         return instancePair.1
     }
 
+    public class var acceptableStatusCodeRange: Range<Int> {
```

Comment:
Why don't you use `[Int]` or `NSSet`, `NSIndexSet` (or `Set<Int> in Swift 1.2`) instead? This `Range` type makes it impossible to use status codes with interval.